### DISCUSSION TITLE
Unable to get on the Discord server

### DISCUSSION THREAD

**Discussion by leemthai** [Textualize/textual#4698] [General]
Hey There,

I've tried a few times to get on the Discord this last couple of months:

https://discord.com/invite/Enf6Z3qhVr

I get a "Unable to accept invite" error every time. Is it just me? :-)

Cheers,
Lee

**Comment by willmcgugan** [comment]
We've had a few reports of this happening. Seems to work now. Is it still broken for you?

**Reply by leemthai** [reply]
Yeah, still broken but not to worry. I'm just starting to use Textualize in earnest so plenty to get my teeth into. I'll research Discord invites a bit later in the process.
